components/MnemonicView.tsx

```diff
@@ -156,17 +155,17 @@ const MnemonicView: FC<MnemonicViewProps> = ({
         </Box>
       </Flex>
       <Flex gap="0.625rem" flexWrap="wrap">
-        {$currentWords.map((word, index) => {
+        {wordsArray.map((word, index) => {
           return (
             <MnemonicInput
-              key={`mnemonic-phrase-view-item-${index}`}
+              key={index}
```

Comment:
FYI, this is a pretty trivial change, but IMO using

```
key={`mnemonic-phrase-view-item-${index}`}
```

is unnecessary since it still creates keys that are all based off the index. In other words, adding the same string to all keys doesn't change the fact they're still based off the index. In this case, since the words array is never sorted and all words come in the same order, using the index as a key doesn't pose any issue.